src/evaluate.cpp

```diff
@@ -369,6 +378,9 @@ namespace {
           : Pt ==   ROOK ? attacks_bb<  ROOK>(s, pos.pieces() ^ pos.pieces(Us, ROOK, QUEEN))
                          : pos.attacks_from<Pt>(s);
 
+#ifdef ANTI
```

Comment:
I must have forgotten that we're setting `ei.pinnedPieces` in this variant.  Strange...